### PR TITLE
Update catalog_controller.rb

### PR DESCRIPTION
**Pull request by karenmajewicz**
This is a draft PR to show work done so far. 

Adds new facet names.

### CODE REVIEW

**Comment by hudajkhan** [comment]
I have been trying these additional fields out with the settings as defined here: https://raw.githubusercontent.com/geoblacklight/geoblacklight/4.x-settings-gbl-1.0/lib/generators/geoblacklight/templates/settings.yml on top of the code currently in the main branch and using this catalog controller file.  I used the "all relationships" json file for data (from the develop-4.x branch) and it all seemed to work fine.  It may be useful to add these changes to the controller after the settings files has also been changed in develop-4.x to match the 4.x-settings-gbl-1.0 branch.  I also compared the original file with the changes here and most of the original facet fields seem represented, the show and index fields seem to also have been updated where need be (e.g. "index_year" as opposed to the earlier "year").  Generally, this looks good.  I may try this with a Solr document with all the fields represented but am comfortable with the catalog controller part being merged.

**Comment by karenmajewicz** [comment]
@ewlarson I saw that you added the Source (Collection) facet back. For Metadata 1.0, that is a string. But, for Aardvark, this value is an ID of another item. How would that work here?

**Comment by ewlarson** [comment]
For whatever reason, the Source facet is required for the source relationship "Browse all 4 records..." sidebar link to work properly. Haven't had a chance to investigate, but BL seems to not process facet queries unless the facets appear in the catalog_controller.rb.

File: lib/generators/geoblacklight/templates/catalog_controller.rb [hunk 1]
```diff
@@ -85,38 +84,36 @@ class CatalogController < ApplicationController
...
+    config.add_facet_field Settings.FIELDS.SPATIAL_COVERAGE, :label => 'Place', :limit => 8
+    config.add_facet_field Settings.FIELDS.CREATOR, :label => 'Creator(s)', :limit => 8
+    config.add_facet_field Settings.FIELDS.PUBLISHER, :label => 'Publisher', :limit => 8
+    config.add_facet_field Settings.FIELDS.PROVIDER, label: 'Provider', limit: 8, partial: "icon_facet"
     config.add_facet_field Settings.FIELDS.FORMAT, :label => 'Format', :limit => 8
-    config.add_facet_field Settings.FIELDS.SOURCE, show: false
+    config.add_facet_field Settings.FIELDS.RESOURCE_TYPE, label: 'Resource Type', :limit => 8
+    config.add_facet_field Settings.FIELDS.SUBJECT, :label => 'Subject', :limit => 8
+    config.add_facet_field Settings.FIELDS.ISO_TOPIC_CATEGORY, :label => 'Theme', :limit => 8
+    config.add_facet_field Settings.FIELDS.SOURCE, :label => 'Collection', :limit => 8
```
Comment (tampakis) [inline lib/generators/geoblacklight/templates/catalog_controller.rb:103]: change to:
`config.add_facet_field Settings.FIELDS.SOURCE, :label => 'Collection', :limit => 8, :show => false`

File: lib/generators/geoblacklight/templates/catalog_controller.rb [hunk 2]
```diff
@@ -85,38 +84,36 @@ class CatalogController < ApplicationController
...
+    config.add_facet_field Settings.FIELDS.GEOREFERENCED, :label => 'Georeferenced', :limit => 3
+
 
     # Have BL send all facet field names to Solr, which has been the default
     # previously. Simply remove these lines if you'd rather use Solr request
     # handler defaults, or have no facets.
     config.add_facet_fields_to_solr_request!
 
+
+#SEARCH RESULTS FIELDS
```
Comment (tampakis) [inline lib/generators/geoblacklight/templates/catalog_controller.rb:113]: I think's it's great that these are included. I recommend indenting it a bit to be inline with the other comments

File: lib/generators/geoblacklight/templates/catalog_controller.rb (hunk 1 above)
Comment (karenmajewicz) [reply lib/generators/geoblacklight/templates/catalog_controller.rb]: Thanks. That worked!

File: lib/generators/geoblacklight/templates/catalog_controller.rb (hunk 2 above)
Comment (karenmajewicz) [reply lib/generators/geoblacklight/templates/catalog_controller.rb]: indented in the latest commit.

**Review by ewlarson — APPROVED** [review]
Thanks for all your hard work on this PR. Glad we got so much feedback, too!

**Pull request merged**